massa-execution-worker/src/execution.rs

```diff
@@ -616,6 +620,59 @@ impl ExecutionState {
         Ok(())
     }
 
+    pub fn execute_denunciation_op(
+        &self,
+        operation: &OperationType,
+        _sender_addr: Address,
+        block_credits: &mut Amount,
+    ) -> Result<(), ExecutionError> {
+
+        const ROLL_COUNT: u64 = 1;
```

Comment:
maybe SLASH_ROLL_COUNT would be a more explicit name

Reply:
Renamed and moved to ExecutionConfig!